proto/zitadel/user/v2beta/query.proto

```diff
@@ -29,6 +29,7 @@ message SearchQuery {
         AndQuery and_query = 12;
         NotQuery not_query = 13;
         InUserEmailsQuery in_user_emails_query = 14;
+        ResourceOwnerQuery resource_owner_query = 15;
```

Comment:
naming question: resource_owner or organization?

Reply:
e.g. in v3 we designed an organizationIdQuery: https://docs-git-user-v2-list-zitadel.vercel.app/docs/apis/resources/user_service_v3/user-service-list-users

Reply:
True, will change it to organizationIdQuery, to keep it consistent